packages/main/src/plugin/extension-loader.ts

```diff
@@ -664,14 +706,17 @@ export class ExtensionLoader {
         });
         this.watcherExtensions.set(extension.id, extensionWatcher);
       }
+      if (!this.getDisabledExtensionIds().includes(extension.id)) {
+        const beforeLoadingRuntime = performance.now();
+        const runtime = this.loadRuntime(extension);
+        const afterLoadingRuntime = performance.now();
 
-      const beforeLoadingRuntime = performance.now();
-      const runtime = this.loadRuntime(extension);
-      const afterLoadingRuntime = performance.now();
+        telemetryOptions['loadingRuntimeDuration'] = afterLoadingRuntime - beforeLoadingRuntime;
 
-      telemetryOptions['loadingRuntimeDuration'] = afterLoadingRuntime - beforeLoadingRuntime;
-
-      await this.activateExtension(extension, runtime);
+        await this.activateExtension(extension, runtime);
+      } else {
+        console.log('Not activating disabled extension (' + extension.id + ')');
```

Comment:
```suggestion
        console.log(`Not activating disabled extension ('${extension.id})`);
```

also should it be more with the id first like
`${extensionId} is not being activated as it is disabled`

Reply:
Fixed. I took a look and all the loader messages are like:
  `blah extension (qualified.name) blah`
and all messages from the extension are like:
  `[name] blah`
so I went with:
  `Extension (${extension.id}) not activated because it is disabled`
to be most consistent with the current loader messages.

(in the future maybe we should change these to be a bit more similar. We still need to clear between what's from the loader and what's from the extension, but would be nice to easily scan for everything related to extension X)